docs/operate/get-started/other-hardware/dependencies.md

```diff
@@ -172,46 +179,50 @@ If you need to maintain the state of your resource, see [(Optional) Create and e
 
 ### Optional dependencies
 
+If optional dependencies do not start, the module will continue to build and reconfigure without them.
```

Comment:
```suggestion
If optional dependencies do not start, the module will continue to build and reconfigure without them. Once the optional dependency constructs successfully, your modular resource will be reconfigured to give access to that optional dependency.
```

Reply:
What does "give access" mean here? Would it just be "access" as in the resource accesses the dependency?

Also, does the optional dependency keep trying to construct at some time interval?
```suggestion
If an optional dependency does not start, the module will continue to build and reconfigure without it.
When an optional dependency constructs successfully, your modular resource reconfigures to access to that optional dependency.
```

Reply:
```suggestion
If an optional dependency does not start, the module will continue to build and reconfigure without it.
`viam-server` reattempts to construct the optional dependency every 5 seconds.
When an optional dependency constructs successfully, your modular resource reconfigures so it can access the optional dependency.
```